pkg/grpc/databroker/fast_forward.go

```diff
@@ -33,18 +32,7 @@ func newFastForwardHandler(ctx context.Context, handler SyncerHandler) SyncerHan
 }
 
 func (ff *fastForwardHandler) update(ctx context.Context, c *ffCmd) {
-	versions := make([]uint64, len(c.records))
-	for i, r := range c.records {
-		versions[i] = r.Version
-	}
-
-	now := time.Now()
 	ff.handler.UpdateRecords(ctx, c.serverVersion, c.records)
-	log.Info(ctx).
```

Comment:
These logs are extremely noisy. I don't think they are useful. Code using the syncer should log records if they want to, but we shouldn't do it here.